network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/advanced_table_block.html

```diff
@@ -2,7 +2,7 @@
 {% load l10n wagtailcore_tags %}
 
 {% block block_content %}
-    <div class="article-table-block table-responsive">
+    <div class="article-table-block tw-overflow-x-auto tw-mx-auto {% if self.wide %} tw-w-full {% else %} tw-w-4/5 {% endif %}">
```

Comment:
The bootstrap class `table-responsive` added the following CSS Rules:

```css
overflow-x: auto;
width: 100%;
```

I am adding the same rules, this time using tailwind classes.

If the table is not set to "wide" then the width is reduced to 80% to match the width of the other article content.

Reply:
Smart!